### ISSUE TITLE
Printing of CreateTableSpaceStmt fails when there is a comment

### ISSUE THREAD

**Issue by bolajiwahab** [lelit/pglast#159]
Printing of CreateTableSpaceStmt fails when there is a comment simply because **CreateTableSpaceStmt** carries a location attribute which is different from the usual node location.

Example:
```sql
CREATE TABLESPACE tbl LOCATION 'directory_path' -- sample tablespace;
```
Location above implies the location of the tablespace which is a string and not an integer. This is different from the usual node location.

pglast fails with below error:
```
if nextc.location <= node_location:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: '<=' not supported between instances of 'int' and 'str'
```

To reproduce:
```bash
echo -e """CREATE TABLESPACE tbl LOCATION 'directory_path' -- sample tablespace;\n;""" | pgpp -C
```

**Comment by lelit** [comment]
Oh, I see!

**Comment by lelit** [comment]
This should be fixed in the upcoming v6.5, thanks for the report!

**Comment by bolajiwahab** [comment]
> This should be fixed in the upcoming v6.5, thanks for the report!

Thank you for the quick resolution.

**Comment by lelit** [comment]
As you may have noticed, v6.5 has been quickly superceded by v6.6, due to a test failure.
